### DISCUSSION TITLE
Can I use Postal without Docker?

### DISCUSSION THREAD

**Discussion by munchyo** [postalserver/postal#3266] [Help with using Postal] [answered]
Hi
I should be installation in no Docker environment.

Can I install without Docker?

If I can install it, How and what files should I download ?

**Comment by Talha5** [comment]
I’m not sure why you’d want to use it without Docker, but I’m guessing you might be thinking of it as a desktop application, which it isn’t.

It’s essentially a Ruby on Rails application that you can install anywhere (refer to the Postal documentation).

**Comment by willpower232** [comment, marked as answer]
If you are familiar with running Ruby on Rails applications then you should be able to download the github repo https://github.com/postalserver/postal and run it however you wish.

You will need to run at least one of each of the commands in the traditional [install repo docker-compose.yaml](https://github.com/postalserver/install/blob/main/templates/docker-compose.v3.yml) file (apart from the runner)

You will probably have to reference [the dockerfile](https://github.com/postalserver/postal/blob/main/Dockerfile) in order to make sure you have all the dependencies not listed in the docs.

It will also be quite difficult to offer support for you so you will have to put in more effort if you are having problems.
